stylesheets/_print.scss

```diff
@@ -19,5 +19,29 @@
     font-size: 12px;
     margin-left: 20px;
   }
+  .side-nav {
+    display: none;
+  }
 
+  // Fixes print layout in Product Get Started pages
+  #get-started-instructions{
```

Comment:
We try to stay away from using IDs to style content because they cannot be used more than once and we want to keep our components reuseable. 

Could you switch this to a class?